src/main/java/com/cloudant/client/api/views/ViewResponse.java

```diff
@@ -140,6 +140,29 @@
     ViewResponse<K, V> previousPage() throws IOException;
 
     /**
+     * Returns an opaque pagination token representing the query for the next page of results, which
```

Comment:
It doesn't represent the query, only the parts of the query required for pagination. This description doesn't gel with the fact that one must recreate the query before supplying the token in getResponse().